### DISCUSSION TITLE
Unable to detect Windows client from my Android14

### DISCUSSION THREAD

**Discussion by balaji2091** [localsend/localsend#1628] [Discussion (English, International)]
I am able to send files from windows client to android phone. However, my Android14 Realme phone is unable to detect Windows client. File transfer is one way from windows to Android but not other way around.

I entered the IP address manually, changed the name, port... Still no results. 

Just pops "Error" at the bottom with no explanation

**Comment by Tienisto** [comment]
Try to change to private network on Windows PC. Or try to disable firewall. For more information. Lookup in the issue tab because lots of people already discussed about this with different methods / strategies
